mdb.py

```
@@ -918,6 +918,100 @@ def get_table_content(db, server, database, table):
         db['cnf']['servers'][server]['conn'].close()
         raise ValueError(e)
 
+
+def get_table_metadata(db, server, database, table):
+    """Return column names, row count, and misc config without fetching row data."""
+    try:
+        db_connect(db, server=server, dictionary=False)
+        cur = db['cnf']['servers'][server]['cur']
+        cur.execute(f"SELECT * FROM {_quote_ident(database)}.{_quote_ident(table)} LIMIT 0")
```

Ruff (0.15.9)

[error] Possible SQL injection vector through string-based query construction

(S608)